src/main/java/com/google/api/codegen/viewmodel/metadata/ReadmeMetadataView.java

```diff
@@ -61,6 +62,11 @@ public String type() {
     return ReadmeMetadataView.class.getSimpleName();
   }
 
+  // Used to make header lines in rst readmes.
+  public List<Character> characters(String str) {
```

Comment:
Also if you're just using it to replace all the characters, then make the function do that, rather than make a list. `String#replaceAll` might help.